.github/actions/import-db/action.yml

```diff
@@ -49,17 +53,21 @@ runs:
           echo "====="
           docker logs $DB5NAME || true
           echo "====="
+          # copy assetstore
+          echo Preparing assetstore
+          docker cp ${{ inputs.ASSETSTORE }} dspace${{ inputs.INSTANCE }}:/dspace/
+          echo "====="
           cd ../
           pip install -r requirements.txt || true
           echo "====="
           cd ./src
           # cleanup resume
           rm __temp/resume/*.json || true
-          python3 repo_import.py --resume=false --config=backend.endpoint=$BEURL --config=db_dspace_7.port=$DB7PORT --config=db_dspace_5.port=$DB5PORT --config=db_utilities_5.port=$DB5PORT --config=input.datadir=$DATADIR/data/ --config=input.icondir=$DATADIR/icon/
+          python3 repo_import.py --resume=false --config=backend.endpoint=$BEURL --config=db_dspace_7.port=$DB7PORT --config=db_dspace_5.port=$DB5PORT --config=db_utilities_5.port=$DB5PORT --config=input.datadir=$DATADIR/data/ --config=input.icondir=$DATADIR/icon/ --assetstore=${{ inputs.ASSETSTORE }}
```

Comment:
Also here. what if ${{ inputs.ASSETSTORE }} is empty? It might even crash, based on parseargs setup.